### DISCUSSION TITLE
discussion

### DISCUSSION THREAD

**Discussion by ahmed-elbaioumy235** [openvinotoolkit/openvino#23818] [General]
Greetings, I'm Ahmed Khaled El-Baioumy, a senior student majoring in Computer Science. Over the past two years, I've delved into the realm of machine learning, dedicating my efforts to various projects spanning natural language processing, computer vision, and generative AI. Despite my familiarity with these domains, venturing into the world of open source is uncharted territory for me. I'm eager to contribute but uncertain about the process. Is there still time to apply? Moreover, I'd appreciate guidance on how to initiate my journey into open source development.

**Comment by rkazants** [comment]
Hi @ahmed-elbaioumy235,

If you would apply to GSoC, I think time is over because candidates already had prerequisite tasks and the deadline for application is today.

If you just want to contribute to OpenVINO, please take one of available good first issue from https://github.com/openvinotoolkit/openvino/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22. Now there is no available but please stay tune and we regularly post new ones or release existing ones executed for long time with no response. We are always glad to new contributors.

Best regards,
Roman
